src/main/resources/application.yml

```diff
@@ -180,18 +180,18 @@ kafka:
       data-type: org.folio.innreach.domain.dto.folio.inventorystorage.InstanceIterationEvent
   backoff:
     interval: ${DEFAULT_INTERVAL:95000}
-    max_failure: ${MAX_FAILURE:360}
+    max_failure: ${MAX_FAILURE:1}
   custom-offset: ${DEFAULT_OFFSET:latest}
   custom-concurrency: ${DEFAULT_CONCURRENCY:2}
 
-initial-contribution:
+contribution:
   async:
-    pool-size: ${INITIAL_CONTRIBUTION_POOL_SIZE:50}
+    pool-size: ${CONTRIBUTION_POOL_SIZE:50}
   scheduler:
-    fixed-delay: ${INITIAL_CONTRIBUTION_SCHEDULER_DELAY:10000}
+    fixed-delay: ${CONTRIBUTION_SCHEDULER_DELAY:10000}
     initial-delay: 300000
-  retry-attempts: ${INITIAL_CONTRIBUTION_RETRIES:3600}
-  fetch-limit: ${INITIAL_CONTRIBUTION_FETCH_LIMIT:50}
-  item-pause: ${INITIAL_CONTRIBUTION_ITEM_PAUSE:0.5}
+  retry-attempts: ${CONTRIBUTION_RETRIES:3600}
+  fetch-limit: ${CONTRIBUTION_FETCH_LIMIT:50}
+  item-pause: ${CONTRIBUTION_ITEM_PAUSE:0.5}
```

Comment:
See my comment [above](https://github.com/folio-org/mod-inn-reach/pull/393/files#r1643156094).

Reply:
retry-attempts is the parameter which is responsible for number of retries. we used this InitialContributionEventProcessor and OngoingContributionEventProcessor class